src/stylesheets/components/_nav.scss

```diff
@@ -59,3 +65,46 @@
     }
   }
 }
+
+.pillbar-fixed {
+  margin: auto;
+  position: fixed;
+  top: 0;
+
+  ul {
+    margin: auto;
+  }
+
+  li {
+    box-shadow: inset 0 -16px 0 rgba(0, 0, 0, 0.1);
+
+    &:first-child, &:last-child {
+      border-radius: 0;
+    }
+  }
+
+  &:first-child, &:last-child {
+    border-radius: 0px;
+  }
+}
+
+.pillbar::before {
+  background-color: $florence-red;
+  border-radius: 0px;
+  box-shadow: inset 0 -16px 0 rgba(0, 0, 0, 0.1);
+  content: '';
+  height: 35px;
+  position: absolute;
+  top: 0;
+  transition: transform 0.2s;
+  width: 100px;
```

Comment:
Did the `calc(100% - 35px)` approach work?

Reply:
I tried it and it took 100% to mean 100% of the page width, since we set .pillbar width: 100% -- 25% works or I can set it to 430px which achieves the same result